packages/components/pages/f-checkout/src/components/_tests/Selector.test.js

```diff
@@ -44,10 +44,10 @@ describe('Selector', () => {
                     propsData
                 });
 
-                const selector = wrapper.find('[data-test-id="formfield-order-time-dropdown-select"]');
+                const selector = wrapper.find('[data-test-id="formfield-order-time-label"]');
```

Comment:
I think it is for now - I'm going to look through the tests I've altered in a separate PR after this in case there's anything else we need to do. But as it's a minor test and the code base is in maintenance mode anyway, happy to merge with this as the current solution 👍